pytest.ini

```
@@ -1,2 +1,10 @@
 [pytest]
-addopts=--flake8
+addopts =
+    --strict-config
+    --strict-markers
+    --flake8
+xfail_strict = True
+junit_family = xunit2
+filterwarnings =
+    error
+    ignore:SelectableGroups dict interface is deprecated\. Use select\.:DeprecationWarning
```

I'd recommend only filtering this warning for flake8.
```suggestion
    # Suppress deprecation warning in flake8
    ignore:SelectableGroups dict interface is deprecated::flake8
```